frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -124,57 +119,57 @@ action.GitMachete.ISyncToRemoteStatusDependentAction.description.sync-to-remote-
+action.GitMachete.CheckoutSelectedBranchAction.undefined.branch-name=Checkout disabled due to undefined branch name
 
 action.GitMachete.CheckoutSelectedBranchAction.description.disabled.currently-checked-out=Branch ''{0}'' is currently checked out
 action.GitMachete.CheckoutSelectedBranchAction.description.precise=Checkout branch ''{0}''
 action.GitMachete.CheckoutSelectedBranchAction.task-title=Checking out
 
 
 action.GitMachete.FastForwardParentToMatchSelectedBranchAction.text=Git Machete: Fast Forward Parent To Match Selected Branch
-action.GitMachete.FastForwardParentToMatchSelectedBranchAction.GitMacheteContextMenu.text=_Fast Forward Parent To Match Branch
+action.GitMachete.FastForwardParentToMatchSelectedBranchAction.GitMacheteContextMenu.text=_Fast Forward Parent To Match This Branch
 action.GitMachete.FastForwardParentToMatchSelectedBranchAction.description=Fast forward parent to match selected branch
 
 
 action.GitMachete.OverrideForkPointOfSelectedBranchAction.text=Git Machete: Override Fork Point of Selected Branch...
-action.GitMachete.OverrideForkPointOfSelectedBranchAction.GitMacheteContextMenu.text=O_verride Fork Point of Branch...
+action.GitMachete.OverrideForkPointOfSelectedBranchAction.GitMacheteContextMenu.text=O_verride Fork Point...
 action.GitMachete.OverrideForkPointOfSelectedBranchAction.description=Select where the range of commits belonging to this branch starts
 
 
 action.GitMachete.PullSelectedBranchFastForwardOnlyAction.text=Git Machete: Pull Selected Branch
-action.GitMachete.PullSelectedBranchFastForwardOnlyAction.GitMacheteContextMenu.text=P_ull Branch
+action.GitMachete.PullSelectedBranchFastForwardOnlyAction.GitMacheteContextMenu.text=P_ull
 action.GitMachete.PullSelectedBranchFastForwardOnlyAction.description=Pull (fast-forward only) this branch
 
 
 action.GitMachete.PushSelectedBranchAction.text=Git Machete: Push Selected Branch...
-action.GitMachete.PushSelectedBranchAction.GitMacheteContextMenu.text=_Push Branch...
+action.GitMachete.PushSelectedBranchAction.GitMacheteContextMenu.text=_Push...
 action.GitMachete.PushSelectedBranchAction.description=Push this branch
 
 
 action.GitMachete.RebaseSelectedBranchOntoParentAction.text=Git Machete: Checkout and Rebase Selected Branch Onto Parent...
-action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.text=Checkout and _Rebase Branch Onto Parent...
+action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.text=Checkout and _Rebase Onto Parent...
 action.GitMachete.RebaseSelectedBranchOntoParentAction.description=Checkout and rebase this branch onto parent
 
 
 action.GitMachete.ResetSelectedBranchToRemoteAction.text=Git Machete: Checkout and Reset Selected Branch to Remote
-action.GitMachete.ResetSelectedBranchToRemoteAction.GitMacheteContextMenu.text=Checkout and Re_set Branch to Remote
+action.GitMachete.ResetSelectedBranchToRemoteAction.GitMacheteContextMenu.text=Checkout and Re_set to Remote
```

Comment:
Hmm maybe:
```suggestion
action.GitMachete.ResetSelectedBranchToRemoteAction.GitMacheteContextMenu.text=(Checkout and) Re_set to Remote
```
?

Reply:
No, it's not necessary coz it's only displayed when a selected branch is not a current branch